maas/resource_maas_vm_host.go

```diff
@@ -303,9 +304,29 @@ func deployMachineAsVMHost(ctx context.Context, client *client.Client, machineId
 		return nil, err
 	}
 
+	// Get Default OS and series
+	var defaultOsystem string
+	defaultOsystemBytes, err := client.MAASServer.Get("default_osystem")
+	if err != nil {
+		fmt.Println(err)
+	}
+	err = json.Unmarshal(defaultOsystemBytes, &defaultOsystem)
+	if err != nil {
+		fmt.Println(err)
+	}
+	var defaultSeries string
+	defaultSeriesBytes, err := client.MAASServer.Get("default_distro_series")
+	if err != nil {
+		fmt.Println(err)
+	}
+	err = json.Unmarshal(defaultSeriesBytes, &defaultSeries)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Maybe instead printing an error  we should return `nil, err`?
Or will it eventually fail because it'll try to use emtpy OS and distro?
 ```go
DistroSeries:   fmt.Sprintf("%s/%s", defaultOsystem, defaultSeries)
```

Reply:
you are right. The fmts were leftovers from a playground code that I maintain. I replaced them with returns. Good catch :+1: